app/build.gradle

```diff
@@ -105,8 +86,40 @@ android {
             it.buildConfigField "String", "SIGNING_KEY", "\"\""
             it.buildConfigField "String", "BUILD_YEAR", "\"${buildYear}\""
         }
+
+        release {
+            minifyEnabled true
+            proguardFile 'proguard-files/proguard-rules.pro'
+            proguardFile 'proguard-files/proguard-butterknife.pro'
+            proguardFile 'proguard-files/proguard-glide.pro'
+            proguardFile 'proguard-files/proguard-google-play-services.pro'
+            proguardFile 'proguard-files/proguard-gson.pro'
+            proguardFile 'proguard-files/proguard-maps-utils.pro'
+            proguardFile 'proguard-files/proguard-prettytime.pro'
+            proguardFile 'proguard-files/proguard-rxjava.pro'
+            proguardFile 'proguard-files/proguard-sentry.pro'
+            proguardFile 'proguard-files/proguard-square-retrofit.pro'
+            proguardFile 'proguard-files/proguard-support-v7-appcompat.pro'
+            proguardFile getDefaultProguardFile('proguard-android.txt')
+            signingConfig signingConfigs.flowRelease
+        }
         debug {
             ext.enableCrashlytics = false
+            minifyEnabled true
+            multiDexEnabled true
```

Comment:
Enabled multidex for debug builds only as we reached 65k method limit but in release we are fine thanks to proguard